apps/super-app/src/App.test.tsx

```diff
@@ -3,27 +3,30 @@ import { render, screen } from '@testing-library/react';
 import App from './App';
 import { MemoryRouter } from 'react-router-dom';
 
-test('renders a nonsense homepage', () => {
+test('it should render support cards on /', () => {
   render(
     <MemoryRouter>
       <App />
     </MemoryRouter>
   );
-  const todoLink = screen.getByText(/TODO/i);
-  expect(todoLink).toBeInTheDocument();
+  const header = screen.getByText('Free Support Services');
+  expect(header).toBeInTheDocument();
+  const twc2 = screen.getByText('Transient Workers Count Too');
+  expect(twc2).toBeInTheDocument();
+  const pbsg = screen.getByText('Pro Bono SG');
+  expect(pbsg).toBeInTheDocument();
 });
 
-test('renders some nonsense buttons on /scratch', () => {
+test('it should render support cards on /landing-page', () => {
```

Comment:
Ah, I didn't mean that we should prepend the names with `it`, I meant that we should write the name _as if_ there was an "it" in front. So in these cases, the fix would be to remove the "it".

Reply:
Oh right now I rmb, the Angular Mocha/Jasmine syntax was `it(_______)`